src/components/Form.js

```diff
@@ -1,10 +1,15 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { loginAction } from '../actions/gameAction';
```

Comment:
O que acha de pular uma linha entre as importações de bibliotecas e de componentes? 😄 
```suggestion
import { connect } from 'react-redux';

import { loginAction } from '../actions/gameAction';
```